tests/layer_tests/jax_tests/test_iota.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import jax
+import numpy as np
+import pytest
+from jax import numpy as jnp
```

Comment:
please use `np` instead

Reply:
I have already tried np as input_type but it was not working on my end, I also tried various approaches but it was not working with tests.

Reply:
anyway, please use numpy array instead of jax array. I will take a look after it.